### PR TITLE
Add keystore managment guide #256

### PR DESCRIPTION
**Pull request by SaveTheAles**

### CODE REVIEW

File: docs/keystore.md [hunk 1]
```diff
@@ -1 +1,117 @@
...
+
+::: danger Warning
+We strongly recommend _NOT_ using the same passphrase for multiple keys. The cyberd team and the cyber•Congress team will not be responsible for the loss of funds.
+:::
+
+## Keystore
+
+All user keypairs stored locally at `PATH_TO_CYBERD/cli/keys` folder. You can backup this folder for easy migrating between testnets.
+
+The node keypair stored in `priv_validator_key.json` at `PATH_TO_CYBERD/daemon/config` folder. You can delete it and restart docker if you want to change this keypair. The new pair will create automatically. Or you can backup it too and insert in the same folder at new testnet if you want same validator pubkey.
```
Comment (litvintech) [inline docs/keystore.md]: Need to describe node and validators keys separetly

Comment (SaveTheAles) [reply docs/keystore.md]: @litvintech 
You mean two separate docs?

Comment (litvintech) [reply docs/keystore.md]: @SaveTheAles you mentioned only validator key, there is also node key

File: docs/keystore.md [hunk 2]
```diff
@@ -83,7 +85,9 @@ We strongly recommend _NOT_ using the same passphrase for multiple keys. The cyb
 
 All user keypairs stored locally at `PATH_TO_CYBERD/cli/keys` folder. You can backup this folder for easy migrating between testnets.
 
-The node keypair stored in `priv_validator_key.json` at `PATH_TO_CYBERD/daemon/config` folder. You can delete it and restart docker if you want to change this keypair. The new pair will create automatically. Or you can backup it too and insert in the same folder at new testnet if you want same validator pubkey.
+The node keypair stored in `node_key.json` at `PATH_TO_CYBERD/daemon/config` folder. You can delete it and restart docker if you want to change this keypair. The new pair will create automatically. Or you can backup it too and insert in the same folder at new testnet if you want same node pubkey.
```
Comment (SaveTheAles) [inline docs/keystore.md]: @litvintech 
Am I right?

File: docs/keystore.md (hunk 1 above)
Comment (SaveTheAles) [reply docs/keystore.md]: @litvintech I'm not sure that understand you correct. Have made some fixes. Can you take a look now?

**Comment by SaveTheAles** [comment]
@xhipster should be better now

File: docs/keystore.md [hunk 3]
```diff
@@ -1 +1,117 @@
...
+
+## Key types
+
+There are four types of key representations that are used:
+
+- `cyber` a.k.a. address. Is used for linking content, sending liquid tokens, delegating, redelegating, and undelegating tokens to validators. Also, withdrawing rewards and voting.
+  - Derived from account keys generated by `cyberdcli keys add`
+  - e.g. `cyber15h6vd5f0wqps26zjlwrc6chah08ryu4hzzdwhc`
+
+- `cybervaloper` validator address. Is possible to delegate, redelegate and undelegate liquid tokens to/from this address abd invoke staking commands. Used to associate a validator to its operator
```
Comment (xhipster) [inline docs/keystore.md]: Please, split keys into 2 groups:
- agent
- validator

For every group clear way of backup and restore must be provided. Now this 3 things are mixed in a text

Comment (SaveTheAles) [reply docs/keystore.md]: @xhipster add some fixes
take a look please

**Comment by litvintech** [comment]
@SaveTheAles need to mention cosmos hub import and compatibility

**Comment by SaveTheAles** [comment]
> @SaveTheAles need to mention cosmos hub import and compatibility
@litvintech done

**Pull request merged**